fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -197,6 +199,12 @@ public boolean dynamicPartitionExists() {
                 && tableProperty.getDynamicPartitionProperty().isExist();
     }
 
+    public boolean isZOrderSort() {
+        return tableProperty != null
+                && tableProperty.getDataSortInfo() != null
+                && tableProperty.getDataSortInfo().getSortType() != TSortType.LEXICAL;
```

Comment:
```suggestion
                && tableProperty.getDataSortInfo().getSortType() == TSortType.ZORDER;
```

So that we don't need to modify it if we add some other sort type later.